bids_prov/__init__.py

```diff
@@ -1,4 +1,14 @@
 import string
 import random
 
-get_id = lambda: "".join(random.choice(string.ascii_letters) for i in range(10))
+
+random.seed(14) # Control random generation for test, init at each import
```

Comment:
Could you add an option here so that the seed is fixed for testing but random otherwise (i.e. in the general case).